tests/app/notifications/rest/test_send_notification.py

```diff
@@ -1143,13 +1147,17 @@ def test_should_not_allow_email_notifications_if_service_permission_not_set(
     client,
     mocker,
     sample_api_key,
-    sample_template_without_email_permission,
+    sample_service,
+    sample_template,
 ):
     mocked = mocker.patch('app.celery.provider_tasks.deliver_email.apply_async')
 
-    data = {'to': 'notify@digital.cabinet-office.gov.uk', 'template': str(sample_template_without_email_permission.id)}
+    service = sample_service(service_permissions=[SMS_TYPE], check_if_service_exists=True)
```

Comment:
ditto no "service_name"
```suggestion
    service = sample_service(service_permissions=[SMS_TYPE])
```